contentcuration/contentcuration/tests/viewsets/test_contentnode.py

```diff
@@ -638,6 +639,20 @@ def test_update_contentnode_remove_from_extra_fields(self):
         with self.assertRaises(KeyError):
             models.ContentNode.objects.get(id=contentnode.id).extra_fields["m"]
 
+    def test_update_contentnode_add_to_extra_fields_nested(self):
+        user = testdata.user()
+        metadata = self.contentnode_db_metadata
+        contentnode = models.ContentNode.objects.create(**metadata)
+        self.client.force_authenticate(user=user)
+        # Remove extra_fields.options.modality
```

Comment:
I know this is already in our codebase, but why do we want to remove this? Is it to re-add the `QUIZ` modality?

Reply:
Oh, I just copy pasted tests and didn't update the comments.... will fix.

Reply:
Updated!